config/default.yml

```diff
@@ -678,8 +688,10 @@ Lint/UnreachableCode:
 Lint/UnreachableLoop:
   Enabled: false
 
+# TODO: Enable this since it's in the styleguide.
 Lint/UnusedBlockArgument:
   Enabled: false
+  StyleGuide: https://github.com/github/rubocop-github/blob/main/STYLEGUIDE.md#spaces-around-equals
```

Comment:
Should this be https://github.com/github/rubocop-github/blob/main/STYLEGUIDE.md#underscore-unused-vars?

Reply:
Oops, yes it should! Feel free to submit a PR!